anubis/entrypoints/puppet-parser

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# Establish source dir for relative includes
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+# Pre-suite (download and extract requested release)
+# shellcheck source=../shared/entrypoint-pre disable=SC1091
+. "$DIR/../shared/entrypoint-pre"
+
+# Make sure required Anubis env vars are set
+# shellcheck source=../shared/entrypoint-anubis-pre disable=SC1091
+. "$DIR/../shared/entrypoint-anubis-pre"
+
+# TODO: these hardcoded values will be replaced by output from something like
+# eval $(pdk environment --puppet-version=\${\$puppet_version})
+# Once that change is incorporated, we may want to consolidate the output generation and
+# error handling into a shared script.
+export PATH=$PATH:/opt/puppetlabs/pdk/private/ruby/2.5.8/bin:/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0
+export GEM_PATH=/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0:/opt/puppetlabs/pdk/share/cache/ruby/2.5.0:/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0
+
+# Run parser and emit output to json document
+/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0/bin/puppet parser validate --render-as json manifests > parser_output.json || true
+exit_code=$?
+# Ensure file contains valid JSON if there's no output
+if [[ ! -s parser_output.json ]]; then
+  echo 'null' > parser_output.json
+fi
+ruby -e "require 'json'; puts ({ exit_code: $exit_code, output: JSON.parse(File.read('parser_output.json')) }).to_json" > anubis_output.json
```

Comment:
I started in on consolidating most of this into a separate, shared script, but thought it might make sense to wait until the environment determination is updated. But I **think** we could pull all of this into a script that accepts an output command to `eval`. And for entrypoints like `puppet-lint`, there can just be another line to overwrite `anubis_output` to whatever the specific needs are there.